src/common/pf_snmp.c

```diff
@@ -13,21 +13,92 @@
  * full license information.
  ********************************************************************/
 
-#ifdef UNIT_TEST
-#endif
-
 /**
  * @file
  * @brief Helper functions for use by platform-dependent SNMP server (agent).
  *
  */
 
+#ifdef UNIT_TEST
+#define pf_lldp_get_management_address mock_pf_lldp_get_management_address
+#define pf_lldp_get_peer_management_address                                    \
+   mock_pf_lldp_get_peer_management_address
+#define pf_lldp_get_link_status      mock_pf_lldp_get_link_status
+#define pf_lldp_get_peer_link_status mock_pf_lldp_get_peer_link_status
+#endif
+
 #include "pf_includes.h"
 #include <string.h>
 
 #define STRINGIFY(s)   STRINGIFIED (s)
 #define STRINGIFIED(s) #s
 
+static uint8_t pf_snmp_bitswapped_byte (const uint8_t byte)
```

Comment:
I prefer to have documentation on all functions, also the rather trivial ones.

Reply:
Done.